doc/setup.md

```diff
@@ -15,13 +17,55 @@ We suppose (or recommend) that all signers can communicate with each other using
 
 To support for setting up the Tapyrus Signer Network, we provide a command-line utility `tapyrus-setup`
 
-## Step 1. Choose Network ID
+## Tapyrus signer network parameters
+In addition to [Tapyrus network parameters](https://github.com/chaintope/tapyrus-core/blob/master/doc/tapyrus/getting_started.md#tapyrus-network-parameters), Tapyrus signer network uses following unique parameters to each tapyrus network.
 
-Choose Network ID, which is a 32-bit integer, and share among all signers.
+**Node VSS**  
+Node VSS is a sort of Verifiable Secret Share(VSS), which is produced at 4.1 Key Generation Protocol in [the paper](https://doi.org/10.1007/3-540-47719-5_33).
+[The structure](#structure-of-vss) is mentioned later.
+All signers must have Node VSSs from all signers. The VSSs must include its VSS  in federations.toml config file to startup a node.
 
-## Step 2. Generate key pair
+**Node secret share**
+Node secret share is a share that is produced at 4.1 Key Generation Protocol in [the paper](https://doi.org/10.1007/3-540-47719-5_33).
+Each signer must have its Node secret share. 
+This is calculated by collected Node VSSs.
+The value is 32bytes data of private key on secp256k1 curve.
 
-Signer generates node key pairs using `tapyrus-setup createkey`.
+Tapyrus signer network uses the following unique parameters to each block generation round.
```

Comment:
Should be "Tapyrus-signer network" or "Tapyrus Signer Network"

Reply:
I fixed